packages/server/migrations/201908131532000-table-global-settings.js

```diff
@@ -0,0 +1,54 @@
+module.exports = {
+  up: async function (queryInterface, Sequelize) {
+    try {
+      await queryInterface.sequelize.query(`BEGIN`);
+
+      await queryInterface.createTable('GlobalSettings', {
+        id: {
+          allowNull: false,
+          primaryKey: true,
+          type: Sequelize.UUID
+        },
+        name: {
+          type: Sequelize.STRING,
+          allowNull: false,
+          unique: true,
+        },
+        entity: {
+          type: Sequelize.JSON,
+          allowNull: true,
+        },
```

Comment:
If this represents the entirety of data.json, then maybe call this field ``entityGraph`` or similar instead, to make it obvious it's a bunch of entities?

Reply:
Sure, thanks.